en/FlyView/videoOverlay.md

```diff
@@ -0,0 +1,13 @@
+# Video Overlay
```

Comment:
This works for me. Perhaps we should add a note up top that this is in daily build only?

> **Note** This is a daily build feature (not present in QGC v3.5) <!-- delete this note when v3.6 is released -->

And BTW, love the feature.

Reply:
Added the note, but also note that I added it under "daily build new features" in the summary, so it can just be moved where it belongs when we get a new release.